proto/pdpb.proto

```diff
@@ -100,6 +100,11 @@ service PD {
     rpc SetExternalTimestamp(SetExternalTimestampRequest) returns (SetExternalTimestampResponse) {}
 
     rpc GetExternalTimestamp(GetExternalTimestampRequest) returns (GetExternalTimestampResponse) {}
+
+    // Get the minimum timestamp across all keyspace groups from API server
+    // TODO: Currently, we need to ask API server to get the minimum timestamp.
+    // Once we support service discovery, we can remove it.
+    rpc GetMinTimestamp (GetMinTimestampRequest) returns (GetMinTimestampResponse) {}
```

Comment:
We need more sync for this interface. I prefer to rename it to GetMinTS (follow the convention of SyncMaxTS) and returns Timestamp structure.

In tsopb.proto in pingcap/kvproto, I defined:
```
rpc GetMinTS (GetMinTSRequest) returns (GetMinTSResponse) {} 

message GetMinTSResponse {
    ResponseHeader header = 1;
    pdpb.Timestamp timestamp = 2;
}
```
In the client-go, I added the following API to the PD (TSO) client, because GetMinTS is used to replace GetTS in the cases where it needs min ts, so I keeps the same return data type, and it isn't necessary to convert "uint64 timestamp" to (physical, logic int64).
```
	// GetMinTS gets a timestamp from PD or the minimal timestamp across all keyspace groups from
	// the TSO microservice.
	GetMinTS(ctx context.Context) (int64, int64, error)
```

Reply:
If we use GetMinTS here, the client will have two functions with the same name.

Reply:
Don't understand. We do we need two different GetMinTS or GetMinTimestamp? Shouldn't they be the same?